lighthouse-core/audits/audit.js

```diff
@@ -136,7 +137,18 @@ class Audit {
     if (displayValue === score) {
       displayValue = '';
     }
+    let auditDescription = audit.meta.description;
 
+    if((result.error || result.rawValue === false) && audit.meta.fallbackDescription) {
```

Comment:
Yeah, sorry, was to split up the logic abit to get feedback + simplify for initial iteration